source-oracle-flashback/Dockerfile

```diff
@@ -0,0 +1,47 @@
+# syntax=docker/dockerfile:1
+FROM python:3.12-slim as base
+FROM base as builder
+
+ARG CONNECTOR_NAME
+
+RUN apt-get update && \
+    apt install -y --no-install-recommends \
+    python3-poetry
+
+RUN python -m venv /opt/venv
+ENV VIRTUAL_ENV=/opt/venv
+
+WORKDIR /opt/${CONNECTOR_NAME}
+COPY ${CONNECTOR_NAME} /opt/${CONNECTOR_NAME}
+COPY estuary-cdk /opt/estuary-cdk
+
+RUN poetry install
+
+
+FROM base as runner
+
+ARG CONNECTOR_NAME
+ARG CONNECTOR_TYPE
+ARG DOCS_URL
+# The USAGE_RATE arg is required, because GH actions doesn't seem to have a way to conditionally
+# pass it only for the connectors that should have a 0 rate. Comes from `usage_rate` in the
+# `python.yaml` workflow matrix.
+ARG USAGE_RATE
+
+RUN apt-get update && \
+    apt install -y --no-install-recommends \
+    openssh-client
+
+LABEL FLOW_RUNTIME_PROTOCOL=${CONNECTOR_TYPE}
+LABEL FLOW_RUNTIME_CODEC=json
+LABEL dev.estuary.usage-rate=${USAGE_RATE}
+
+COPY --from=builder /opt/$CONNECTOR_NAME /opt/$CONNECTOR_NAME
+COPY --from=builder /opt/estuary-cdk /opt/estuary-cdk
+COPY --from=builder /opt/venv /opt/venv
+COPY --from=ghcr.io/estuary/network-tunnel:dev /flow-network-tunnel /usr/bin/flow-network-tunnel
```

Comment:
Can we add this to copy in the network tunnel into the base python connector image instead? Most connectors won't need it but others in the future probably will and that way they wouldn't all need to have separate docker files and it keeps the github workflow simpler. It doesn't seem like it would be a problem for connectors to have the network tunnel binary backed in to their running dockerfile even if they didn't end up needing it.

Reply:
I think it is okay to add network tunnel to the base image, but I think we will end up needing custom Dockerfiles anyway, so I kept that part but commented it